site/content/en/reference/pkg/fix/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Fix"
+linkTitle: "fix"
+type: docs
+description: >
+   Fix a local package.
+---
+<!--mdtogo:Short
+    Fix a local package.
+-->
+
+Fix reads the local package, modifies the package to use the latest kpt features 
+and fixes any deprecated feature traces
+
+### Examples
+
+#### Example fix commands
+<!--mdtogo:Examples-->
+```sh
+# print the fixes which will be made to the package without actually modifying 
+# resources
+kpt pkg fix . --dry-run
+```
+
+```sh
+# fix the package if it is using deprecated features
+kpt pkg fix .
+```
+<!--mdtogo-->
+
+### Synopsis
+<!--mdtogo:Long-->
+```
+kpt pkg fix LOCAL_PKG_DIRECTORY [flags]
+
+LOCAL_PKG_DIRECTORY:
+  Local directory with kpt package. Directory must exist and
+  contain a Kptfile.
+
```

Comment:
Include information about the `dry-run` flag here.

Reply:
Done.